src/linux/backend-v4l2.h

```diff
@@ -413,6 +416,20 @@ namespace librealsense
             std::atomic<bool> _is_started;
             std::unique_ptr<std::thread> _thread;
             std::unique_ptr<named_mutex> _named_mtx;
+            struct device {
```

Comment:
Most of this struct is not is use.
You only write + read from `type`
and you write to 
```
num_planes
cap
cropcap
```
But don't use it anywhere.
The struct itself is not in use too nowhere.